payments-core/src/test/java/com/stripe/android/networking/StripeApiRepositoryTest.kt

```diff
@@ -1959,6 +1959,7 @@ internal class StripeApiRepositoryTest {
                     assertThat(this["consumer_session_client_secret"]).isEqualTo(clientSecret)
                 }
                 assertThat(this["id"]).isEqualTo(id)
+                assertThat(this["payment_user_agent"].toString()).startsWith("stripe-android/")
```

Comment:
Without the fix, this would fail with.
```
expected to start with: stripe-android/
but was               : (payment_user_agent, stripe-android/20.48.0)
```